packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2609,74 +2539,215 @@ export class LitNodeClientNodeJs
     const responseData = res.values;
     logWithRequestId(
       requestId,
-      'responseData',
+      '[signSessionKey] responseData',
       JSON.stringify(responseData, null, 2)
     );
 
     // ========== Extract shares from response data ==========
     // -- 1. combine signed data as a list, and get the signatures from it
-    const signedDataList = responseData.map(
-      (r: any) => (r as SignedData).signedData
-    );
+    let curveType = responseData[0]?.curveType;
+
+    if (!curveType) {
+      log(`[signSessionKey] curveType not found. Defaulting to ECDSA.`);
+      curveType = 'ECDSA';
```

Comment:
signSessionKey endpoint can't be ECDSA. Replace all the ECDSA logic from below? 
cc: @glitch003

Reply:
This is needed when you want to test all existing V0 networks and V1. A separated PR will be raised to remove all of these once we are ready to merge this in to master